### ISSUE TITLE
Transcribe problem - NullPointerException

### ISSUE THREAD

**Issue by EskildPa** [Audiveris/audiveris#249]
What is wrong with this PDF? 
All in all, I am pretty impressed with Audiveris. I have transcribed several pages without too many corrections, but this page results in no usable MXL.
[The Prayer p5print.pdf](https://github.com/Audiveris/audiveris/files/2845060/The.Prayer.p5print.pdf)
Output from the command window here:
[CmdCopy.txt](https://github.com/Audiveris/audiveris/files/2845084/CmdCopy.txt)

**Comment by museflip** [comment]
Hello,
To whoever developper who wants to tack this issue further, measure 34 in piano part leads to the error.
Enclosed : score which process is OK (piano measure 34 replaced by 35)
Sincerely
![the prayer p5print_ok_process](https://user-images.githubusercontent.com/46139401/52526473-78df9000-2cb9-11e9-8b79-6206b496e89d.jpg)

**Comment by Bacchushlg** [comment]
The problem is mostly: not recognized augmentation dots. This applies to measures 34 and 35.
Additionally the "e" in the bass part of piano (measure 34) is recognized as 2 note heads (of two voices) (can be seem by the slash across). This leads to a rhythm miss-match. Finally several "b" at the stave line beginning (for F-Dur) are not recognized. This leads to wrong notation, of course.
I append the omr-File after correction:

[The.Prayer.p5print.omr.zip](https://github.com/Audiveris/audiveris/files/2860902/The.Prayer.p5print.omr.zip)

**Comment by EskildPa** [comment]
@Bacchushlg Thank you so much, after fixing the "b"s, the result in Musescore was fine.
I tried to do these corrections myself, but couldn't get correct rhythm for measures 34 and 35.
![image](https://user-images.githubusercontent.com/47450103/52740856-480e9c00-2fd4-11e9-83ec-6e5846dc0aa7.png)
I assume the problem is related to voice assigning, but I am not able to get i right. Any hints?

I didn't expect these errors to result in a NullPointerException, as missing augmentation dots are quite common. But I can't remember to have seen any slash on a notehead before. Maybe there is a combination of the mentioned errors that throws the exception?

**Comment by Bacchushlg** [comment]
curious. I get a slash through the "e" (the high one, with dot).
Try to delete the head. In my case a second head "behind" appear and the measure is correct.
Also I don't get the last measure as wrong (rose).

Do we use the same version? Mine is 5.1.0:ff26ab430

**Comment by EskildPa** [comment]
Yes, I found the "e" with the slash, and deleted the 2 heads. The picture shows the state after I did as many corrections as possible, sorry I didn't mention that.

My Audiveris is 5.1.0:4687c3527 (Windows installer 13 Dec 2018 64bit).

**Comment by hbitteur** [comment]
[Sorry, I'm not really available right now because of many home works going on.]

But I just ran the case that indeed leads to NPE.
The root cause is the fact that beam 1245 and stem 3550 got connected, because the vertical gap between them is not very high (and may happen in poor quality scores).
This resulted in having beams 1245/1252 and 1247/1249 in the **same** beam group.
Inconsistency was detected and a "split" was attempted...

I have fixed the NPE, by avoiding the removal of a null relation (which caused the NPE), and thus the transcription goes to the end.
But this is just a quick fix.

Available in "development" branch, commit a840428d7f0ceb5b43953c5d74e936df5109d43e
